website/content/en/docs/concepts/disruption.md

```diff
@@ -327,3 +321,7 @@ spec:
     budgets:
       - nodes: "0"
 ```
+
+{{% alert title="Note" color="primary" %}}
+Unlike other form of disruption, Karpenter will drift nodes by launching one replacement node at a time despite nodepool budget allowing for more then one node. The process will wait until the replacement node is ready and initialized before beginning to launch a new node. Karpenter disrupts and upgrades your nodes, prioritizing availability and safety to your applications.
```

Comment:
I almost think that we shouldn't include this. This is a bit misleading given what we actually do. We allow more nodes to be disrupted while we are waiting for that other node to go initialized -- up to the budget

Reply:
I sort of agree. I think we want to let users know that we enqueue nodes for disruption, but be clear that we're not only going to do one at a time all the time.